add note on other SDKs (#119)
temporalio/documentation: docs/other-sdks.md

---
id: other-sdks
title: Other SDKs
sidebar_label: Other SDKs
---

> There is demand for other Temporal SDKs other than Go and Java. The user community has taken initiative to build some of these, but please note that they are not officially supported or endorsed by Temporal unless explicitly mentioned:

Reviewer: - Paragraph could be more direct, consider:

Looking for SDKs in other languages? There are plans to support many other SDKs, but currently we only support Go and Java. This is a list of ones that are in the works but are not yet officially endorsed:

- In the context of the list, I am not sure what it means to be explicitly mentioned? I imagine that officially supported SDks would have a spot on the root of our docs pages, so anything still on this list wouldn't be officially endorsed?

Author: not officially endorsed yes, but the vision for why we explicitly mention is to point people toward "good enough" third party SDKs. obv a lot of placeholders now as we work it out.

Author: ```suggestion
There is demand for other Temporal SDKs other than Go and Java. The user community has taken initiative to build some of these, but please note that they are not officially supported or endorsed by Temporal unless explicitly mentioned:
```

> - [Ruby SDK by Coinbase](https://github.com/coinbase/temporal-ruby)

Reviewer: Should we be linking to these? Is there any other context we can provide here about the status ?

Author: i dont know but i think it looks good to link. since everything on this page will not be "officially supported" but we can still acknowledge the work being done by the community. @rylandg ?

Reviewer: Cool - sgtm then. Just curious if we plan to fork it and maintain?

Author: for Ruby, no idea, but i'd guess its secondary compared to python/nodejs (since we probably trust Coinbase's impl)

- Python SDK - please get in touch with us
- Swift SDK - please get in touch with us

> - C#/dotNet SDK - please get in touch with us

Reviewer: C#/.Net ?

- Node.js SDK - please get in touch with us
- Rust SDK - please get in touch with us
- C++ SDK - please get in touch with us

> 

Reviewer: - [PHP](https://github.com/temporalio/php-sdk) ?

Author: ```suggestion
- [PHP](https://github.com/temporalio/php-sdk) - maintained by Temporal, but not officially launched, please get in touch
```

> We are very excited by the interest of the community, but please don't go too far without getting in touch with us! Collaboration via open source is something we strongly believe in, but we also care about the long term developer experience and therefore likely will want to take on maintenance leadership of SDKs in key languages.

Reviewer: I think this message could be more focused. To me it seems like the key message we want to communicate here is that we want people to work with us. I am not sure if we want to say that we want to take ownership directly here...Consider:

If you are interested in using an unofficial SDK, we recommend that you reach out to any of us at Temporal. We are very excited by the community's enthusiasm and strongly believe in collaborating on open source projects. Our goal is to make sure there is maintenance leadership in place to care for the developer experience over the long term.

Author: ```suggestion
If you are interested in using an unofficial SDK, we recommend that you reach out to any of us at Temporal. We are very excited by the community's enthusiasm and strongly believe in collaborating on open source projects. Our goal is to make sure there is maintenance leadership in place to care for the developer experience over the long term.
```
